docker/supabase/.env.example

```diff
@@ -0,0 +1,41 @@
+# Secrets
```

Comment:
You can delete this file now, or leave a comment in here to see `config/env.development` and the other files.